server/auth/sso/sso.go

```diff
@@ -65,7 +65,15 @@ type Config struct {
 	RedirectURL  string                  `json:"redirectUrl"`
 	RBAC         *rbac.Config            `json:"rbac,omitempty"`
 	// additional scopes (on top of "openid")
-	Scopes []string `json:"scopes,omitempty"`
+	Scopes  []string        `json:"scopes,omitempty"`
+	Timeout metav1.Duration `json:"timeout,omitempty"`
```

Comment:
I wonder if we should call this something else? It's not really a timeout - its more like a session expiry